src/validation.h

```diff
@@ -289,7 +289,8 @@ class CScriptCheck
 
     bool operator()();
 
-    void swap(CScriptCheck &check) {
+    void swap(CScriptCheck& check) noexcept
```

Comment:
I think we should add it to our code in `hdchain.h` also:

```
-void swap(CHDChain& first, CHDChain& second) // nothrow
+void swap(CHDChain& first, CHDChain& second) nothrow
```

Reply:
resolved